packages/components/src/components/hds/form/field/index.hbs

```diff
@@ -26,7 +26,9 @@
     )
   }}
   <div class="hds-form-field__control">
-    {{yield (hash Control=(component "hds/yield") id=this.id ariaDescribedBy=this.ariaDescribedBy)}}
+    {{yield
+      (hash Control=(component "hds/yield") id=this.id labelId=this.labelId ariaDescribedBy=this.ariaDescribedBy)
```

Comment:
> since also the id itself is already forwarded, would make sense to just build the value label-[controlId] directly in the SuperSelect, where is actually needed? 

the reason I did it like this (constructing it in the form and passing it to the SuperSelect) was because I wanted to avoid adding a backing class only to concatenate `"label-"` to `id` – I will do that thought, it probably make things easier in the long run